src/datalink/bpf.rs

```diff
@@ -19,17 +19,17 @@ use std::sync::Arc;
 use bindings::bpf;
 use packet::Packet;
 use packet::ethernet::{EthernetPacket, MutableEthernetPacket};
-use datalink::DataLinkChannelType;
+use datalink::{DataLinkChannelType,DataLinkSender,DataLinkReceiver,DataLinkChannelIterator};
 use datalink::DataLinkChannelType::{Layer2, Layer3};
 use internal;
 use util::NetworkInterface;
 
 // NOTE buffer must be word aligned.
-pub fn datalink_channel(network_interface: &NetworkInterface,
+pub fn datalink_channel_bpf(network_interface: &NetworkInterface,
```

Comment:
There's no need for the `_bpf` and `Bpf` suffixes to the various functions and structures - the module system can handle this.